test/test_backend/test_backend.py

```diff
@@ -261,10 +261,12 @@ def test_get_cset_members_items__cols(self):
         self.assertEquals(csmi, expected)
 
     # TODO: test is failing. fix
-    @unittest.skip("Skipping failing/erroring test temporarily.")
+    #  - The cuase of the err is that subgraph() is a different function now. it used to call connected_subgraph_from_nodes(), which no longer exists.
+    #    Is this test still valid? Is there any func that takes a list of concept IDs and returns their parentage tuples?
+    @unittest.skip("May need to remove this test. I think the code it was testing got removed. Skipping failing/erroring test temporarily.")
```

Comment:
## `test_subgraph()` no longer valid? Remove?
@Sigfried I need to know if there's any code that is still relevant to this test, otherwise we need to remove. It used to call `subgraph()`, which called `connected_subgraph_from_nodes()`. I can't find any function like that anymore, a func that takes concept IDs like `[1738170, 1738171, 1738202, 1738203]` and returns tuples like `[ ( "1738170", "1738171" ), ... ]`.